src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageSearch/JsonFormat/NormalVerbosityJsonFormatSearchResultPackage.cs

```diff
@@ -6,17 +6,17 @@
 
 namespace NuGet.CommandLine.XPlat
 {
-    internal class NormalVerbosityPackage : MinimalVerbosityPackage
+    internal class NormalVerbosityJsonFormatSearchResultPackage : MinimalVerbosityJsonFormatSearchResultPackage
```

Comment:
nit: Can you help me understand why this inheritance structure is needed? 

My thinking: 
I agree that these types build upon one another (it's probably unlikely we'd ever need to add/remove a property from a lower level, say, `Minimal` formatting, which would actually need to be available in a higher level formatting, like `Detailed`.

However, is anything gained by this structure besides avoiding copy/pasting code for the properties? I can't think of how testing would be easier, or how developing with these classes would be clear. (I'd still have to validate whether I had a NormalVerbosityJson object and then know which properties that gains me).

One suggestion would be to just keep a reference to your `IPackageSearchMetadata` and then only expose the properties that are relevant. No inheritance would be necessary..